prisma/migrations/20220407201706_init/migration.sql

```diff
@@ -0,0 +1,57 @@
+-- CreateTable
+CREATE TABLE "dashboards" (
+    "id" TEXT NOT NULL,
+    "title" TEXT NOT NULL,
+    "description" TEXT NOT NULL,
```

Comment:
Not sure if there are any platforms for which description can be null.

Reply:
I went with `not null` here because there's usually no difference between `null` and `""`. Allowing nulls for strings usually just leads to bugs later on when you check for empty and forget to check for null, or vice versa.